src/Sylius/Behat/Context/Api/Shop/HomePageContext.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Behat\Context\Api\Shop;
+
+use Behat\Behat\Context\Context;
+use Sylius\Behat\Client\ApiClientInterface;
+use Sylius\Behat\Client\ResponseCheckerInterface;
+use Sylius\Behat\Service\SharedStorageInterface;
+use Symfony\Component\HttpFoundation\Request as HttpRequest;
+use Webmozart\Assert\Assert;
+
+final class HomePageContext implements Context
+{
+    /** @var ApiClientInterface */
+    private $productsClient;
+
+    /** @var ResponseCheckerInterface */
+    private $responseChecker;
+
+    /** @var SharedStorageInterface */
+    private $sharedStorage;
+
+    public function __construct(
+        ApiClientInterface $productsClient,
+        ResponseCheckerInterface $responseChecker,
+        SharedStorageInterface $sharedStorage
+    ) {
+        $this->productsClient = $productsClient;
+        $this->responseChecker = $responseChecker;
+        $this->sharedStorage = $sharedStorage;
+    }
+
+    /**
+     * @When I check latest products
+     */
+    public function iCheckLatestProducts(): void
+    {
+        $this->productsClient->customAction('shop/products/latest', HttpRequest::METHOD_GET);
```

Comment:
Are you sure about this path?

```suggestion
        $this->productsClient->customAction('new-api/shop/products/latest', HttpRequest::METHOD_GET);
```